packages/merge-styles/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "api-extractor": "node ../../scripts/api-extractor.js",
     "build": "node ../../scripts/build.js",
```

Comment:
Is this up to individual engineer to run on each checkin?

Reply:
Once the api-extractor task is enabled for all desired packages, there will be a separate PR to add api-extractor to our CI build.